File: challenges/2-tabuada/c++/lucashe4rt/tabuada.cpp
#include <iostream>

int main()
{
    int numeroFinal;

    std::cout << "Insira um número: ";
    std::cin >> numeroFinal;

    for(int i = 1; i <= numeroFinal; i++) {
    
        std::cout << "===========================" << std::endl;
        std::cout << "A tabuada do número " << i << " é:" << std::endl;
    
        for (int j = 1; j <= 10; j++) {
            std::cout << "" << i << " X " << j << " = " << (i*j) << std::endl; 
        }
    }

    std::cout << "===========================" << std::endl;
}

File: challenges/2-tabuada/js/romariocoimbrac/tabuada.js
function tabuada(numeroFinal) {
  for (let i = 1; i <= numeroFinal; i++) {
    console.log(`A tabuada de ${i} é: `);
    for (let j = 1; j <= 10; j++) {
      console.log(`${i} x ${j} = ${i * j}`);
      console.log(`===========================`);
    }
  }
}

tabuada(2);
